VideoApi/VideoApi/Services/VirtualRoomService.cs

```diff
@@ -111,14 +111,12 @@ private async Task<ParticipantRoom> RetrieveJudicialRoomForConference(Guid confe
             return await GetUpdatedRoom(conference, roomId);
         }
 
-        private string GetIngestUrl(Conference conference, VirtualCourtRoomType roomType, KinlyRoomType kinlyRoomType, long roomId)
+        private string GetIngestUrl(Conference conference, KinlyRoomType kinlyRoomType, long roomId)
         {
-            if (roomType == VirtualCourtRoomType.JudicialShared)
-                return null;
+            if (kinlyRoomType == KinlyRoomType.Interpreter)
+                return $"{conference.IngestUrl}{InterpreterSuffix}{roomId}";
 
-            return kinlyRoomType == KinlyRoomType.Interpreter ?
-                    $"{conference.IngestUrl}/{roomId}/{InterpreterSuffix}" :
-                    $"{ conference.IngestUrl}/{ roomId}";
```

Comment:
We used to return $"{ conference.IngestUrl}/{ roomId}"; for most cases. Do we not definitely want to return null for all other kinlyRoomType now?